assets/components/Dropdown.tsx

```diff
@@ -38,28 +48,39 @@ export function Dropdown({children, isActive, icon, optionLabel, label, value, c
         >
             <button
                 type="button"
-                className={classNames(
-                    'nh-dropdown-button',
-                    {
-                        'nh-dropdown-button--active': isActive,
-                        'nh-dropdown-button--text-only': textOnly,
-                        'nh-dropdown-button--small': small,
-                        'nh-dropdown-button--stretch': stretch,
-                    }
-                )}
+                className={classNames({
+                    'nh-dropdown-button--borderless': borderless,
+                    'nh-dropdown-button': !borderless,
```

Comment:
I wonder why this shouldn't be always there, seems like `--borderless` is modifier but the component itself would be missing the main class without it

Reply:
right, it should always be there